src/frontend/src/components/headerComponent/index.tsx

```diff
@@ -50,11 +49,9 @@ export default function Header(): JSX.Element {
   const routeHistory = useLocationStore((state) => state.routeHistory);
 
   const profileImageUrl =
-    `${BACKEND_URL.slice(
-      0,
-      BACKEND_URL.length - 1,
-    )}${BASE_URL_API}files/profile_pictures/${userData?.profile_image}` ??
-    profileCircle;
+    `${BASE_URL_API}files/profile_pictures/${
```

Comment:
Hi @nicoloboschi,

I appreciate the refactor that removes the code from the src tag in the HTML. These kind of refactor are always very welcome. 

I just wanted to provide some context and ask a question regarding the variable BACKEND_URL.
This variable was created because Langflow's frontend and backend do not always run on the same server URL. Hence, BACKEND_URL was introduced to handle this scenario.
If we revert to using BASE_URL_API, do you think it might break this setup? What are your thoughts?

Reply:
Hey @Cristhianzl 
the scenario is exactly that one, backend and frontend as separate service.
Removing the host will make the browser to reuse the same base host of the current webpage. 

since this is an API call the correct flow is:
- call <frontend_service>/api/....
- The frontend redirects the call to the backend (as it does for other calls)

I've verified the fix with different ports/host for backend and frontend and it works well.

Reply:
Awesome, I will trust in you in that case hahah
Thanks for your explanation and reply!

LGTM